src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -154,7 +154,7 @@ const EmbeddingContinuousIndex = ({ experimentId }) => {
                   experimentId={experimentId}
                   config={config}
                   plotUuid={plotUuid}
-                  plotData={geneExpression.data[shownGene]?.expression}
+                  plotData={geneExpression.data[shownGene]?.truncatedExpression.expression}
```

Comment:
Do we not want to display the `zScore` here sometimes too?

Reply:
I don't know if there was a bug in here, but I do know that if the code was accessing `.expression` then the direct translation to the new structure is making it access the gene expression. 

If a part of this displayed the zScore then it should have referenced `zScore` somewhere.